animated-transformer/src/lib/tokens/token_gemb.ts

```diff
@@ -271,6 +281,25 @@ export function singleNextTokenIdxOutputPrepFn(
   );
 }
 
+// Returns the one Hot representation for each token of the expected output sequence for the provided input sequence
+ export function expectedOutputSeqPrepFn(
+  model: { config: { tokenRep: BasicTaskTokenRep } },
+  inputSeqs: string[][],
+  expectedOutputs: string[][],
+): GTensor<'batch' | 'pos' | 'tokenId'> {
+  // Compute Token rep for inputSeq
+  const inputSeq = inputSeqs.map((SingleSample) => SingleSample.map((token) => model.config.tokenRep.tokenToIdx[token]))
```

Comment:
Style nit: `SingleSample` is a value, so it should start with a lower-case variable. I would name `SingleSample` as `inputSeq` (because its a single example), and I would name what you currently call `inputSeq` as `batchInputs` or something like that (because it is not a single example!).